test/tests/functional/pbs_reservations.py

```diff
@@ -272,3 +272,71 @@ def test_resv_excl_future_resv(self):
         rid2 = self.server.submit(r2)
 
         self.server.expect(RESV, exp_attr, id=rid2)
+
+    def test_resv_excl_with_jobs(self):
+        """
+        Test to see that exclusive reservations in the near term do not
+        interfere with longer term reservations with jobs inside
+        """
+        a = {'resources_available.ncpus': 1}
+        self.server.manager(MGR_CMD_SET, NODE, a, id=self.server.shortname)
+
+        now = int(time.time())
+        a = {'Resource_List.select': '1:ncpus=1',
+             'Resource_List.place': 'excl', 'reserve_start': now + 30,
+             'reserve_end': now + 300}
```

Comment:
30 seconds seems more than necessary. Can we reduce this to 10 or 15 seconds?

Reply:
Hi Mike, I would object to reduce start time any less than 30s due to race conditions we have seen in past.

Reply:
I chose 30s because of what @anamikau said.  We don't want this test to fail on slower machines.